presto-raptor/src/test/java/com/facebook/presto/raptor/integration/TestRaptorIntegrationSmokeTest.java

```diff
@@ -78,6 +78,14 @@ public void testCreateArrayTable()
         assertUpdate("DROP TABLE array_test");
     }
 
+    @Test
+    public void testCreateTableUnsupportedType()
+    {
+        // Need to look into create table as with unsupported types as well.
+        // assertQueryFails("CREATE TABLE rowtype_test AS SELECT row(1) AS c", "Failed to perform metadata operation");
```

Comment:
I think you forgot to uncomment your test :)

Reply:
No. It still doesn't work. Created a new task for that - T48764628

Reply:
I think it's working. Just the failure message in the test expectation is not right.

Reply:
Yes.